packages/manager/modules/pci/src/components/project/deployment-mode-selector/deployment-mode.controller.js

```diff
@@ -0,0 +1,37 @@
+import {
+  DEPLOYMENT_MODE_CHIP_CLASS,
+  DEPLOYMENT_MODE_CHIP_LABEL,
+} from './deployment-mode.constant';
+import { DEPLOYMENT_MODES_URL } from '../../../projects/project/project.constants';
+
+export default class DeploymentModeController {
+  /* @ngInject */
+  constructor(PciProjectDeploymentMode, coreConfig) {
+    this.PciProjectDeploymentMode = PciProjectDeploymentMode;
+    const { ovhSubsidiary } = coreConfig.getUser();
+    this.deploymentModeUrl =
+      DEPLOYMENT_MODES_URL[ovhSubsidiary] || DEPLOYMENT_MODES_URL.DEFAULT;
+  }
+
+  $onInit() {
+    this.deploymentMode = null;
+    this.isLoading = true;
+    this.deploymentModes = null;
+
+    this.load();
+  }
+
+  load() {
```

Comment:
Question: if an error is thrown, `this.isLoading` will be forever truthy ?

Reply:
Yes, there is no error handling at this point

Reply:
Is it planned to improve that behaviour ?

Reply:
No since we will migrate it to react soon